manuscript/00-The-Mythical-100-Percent-Test-Coverage.md

```diff
@@ -20,5 +20,5 @@ you will ***only write*** the code required ***to past the test***(*s*)
 Achieving 100% test coverage is only an issue for a project that
 has not been written with good discipline from the start.
 
-We will cover strategies for how to [***retrofit***](http://en.wikipedia.org/wiki/Patch_(computing))
+We will cover strategies for how to [***retrofit***](http://en.wikipedia.org/wiki/Patch_(computing)
```

Comment:
Hey @ncwhitehead! In actual fact, this last bracket was required as it's closing out the markdown which creates the hyperlink for the word 'retrofit' :blush: Could you add a commit to put that bracket back and then I'll be happy to merge the rest of your amendments? :+1: